app/app.g.less

```diff
@@ -240,7 +240,7 @@ a,
   right: calc(~'100% + 20px');
   align-self: center;
   display: flex;
-  z-index: 10;
+  z-index: 1011;
```

Comment:
hmmm i don't know if we should be doing this on a global class like this, if there isn't a locally scoped class for that element we can just add it to the style itself. also looks like your linter is disagreeing with the team codestyles here for some reason 🤔